code/src/Components/Overview.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import './Overview.css'
+
+const Overview = ({ nameInput, days, where }) => {
+    return (
+        <section className="summary">
+            <h2>Summering</h2>
+            <p style={{color: 'purple'}}>Namn : {nameInput}</p>
```

Comment:
Inline styling is convinient in this case. It's not recommended. Thus keep all the styling in seperate CSS file, give the element a className and style that class in CSS. By this way, it will be easier to maintain and update your page